test/transformation/input/dbs.csv

```diff
@@ -1,4 +1,5 @@
 inr;iso;nam;plz;ort;gemeindekennzahl;sbi;isil;tvw;tel;fax;ema;url;opa;text;oef;typ_text;utr_text;gro_text;leitung
-AB038;DEU;Universitäts- und Stadtbibliothek Köln;Universitätsstr. 33;Köln;53150000000;38;DE-38;-221;4 70-3335 und 2374;(0221) 4 70-5166;sekretariat@ub.uni-koeln.de;www.ub.uni-koeln.de;NULL;aktiv;NULL;NULL;Land;1.000.001 und mehr;nebenamtlich
-AK001;DEU;Staatsbibliothek zu Berlin - Preußischer Kulturbesitz;Potsdamer Str. 33;Berlin;110000000000;1a;DE-1a;-30;2 66-43 25 00;NULL;info@sbb.spk-berlin.de;http://www.staatsbibliothek-berlin.de;http://stabikat.de;aktiv;NULL;Nationalbibliothek;Körperschaft/Stiftung des öff. Rechts;1.000.001 und mehr;nebenamtlich
-AB294;NULL;Ruhr-Universität Bochum. Universitätsbibliothek;Universitätsstr. 150;Bochum;59110000000;294;DE-294;-234;32-22350;(0234) 32 14213;direktion-ub@rub.de;http://www.ub.rub.de;NULL;aktiv;Mo-Fr 8-24 Uhr, Sa 11-20 Uhr, So 11-18 Uhr;NULL;Land;1.000.001 und mehr;nebenamtlich
\ No newline at end of file
+AB038;DEU;Universit�ts- und Stadtbibliothek K�ln;Universit�tsstr. 33;K�ln;53150000000;38;DE-38;-221;4 70-3335 und 2374;(0221) 4 70-5166;sekretariat@ub.uni-koeln.de;www.ub.uni-koeln.de;NULL;aktiv;NULL;NULL;Land;1.000.001 und mehr;nebenamtlich
```

Comment:
There seems to be an encoding issue here. In the diff, correct umlauts are replaced with broken ones.